components/claim/components/error_label.tsx

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+type ErrorLabelProps = {
+    errorText?: string | JSX.Element;
+}
+
+const ErrorLabel = React.memo(({errorText}: ErrorLabelProps) => (errorText ? (
+    <div className='form-group has-error'>
+        <label className='control-label'>{errorText}</label>
+    </div>
+) : null
+));
+
+export default ErrorLabel;
```

Comment:
great work ... just two minor things to change here:

1. please add the `memo` into the default export line. This is important since the component won't have a displayName when we memoize it directly in the declaration (helpful for snapshots)
2. we do not need the wrapping `()`

```suggestion
const ErrorLabel = ({errorText}: ErrorLabelProps) => errorText ? (
    <div className='form-group has-error'>
        <label className='control-label'>{errorText}</label>
    </div>
) : null;

export default React.memo(ErrorLabel);
```

Reply:
Good to know about `memo`. For `()`, I had to keep it because Eslint was complaining about: `Arrow function used ambiguously with a conditional expression.`

Reply:
yeah .. i always forget about that rule... fair enough! :D